sklearn/linear_model/tests/test_logistic.py

```diff
@@ -565,6 +565,38 @@ def test_ovr_multinomial_iris():
         assert_equal(scores.shape, (3, n_cv, 10))
 
 
+def test_ovr_multinomial_iris_binary():
```

Comment:
This seems like a valid regression test but I feel something like this would be simpler:
```py
import numpy as np

from sklearn.linear_model import LogisticRegression
from sklearn.datasets import make_classification

X, y = make_classification()
clf = LogisticRegression(multi_class='multinomial', solver='saga')
clf.fit(X, y)
decision = clf.decision_function(X)
proba = clf.predict_proba(X)
expected_proba_class_1 = (np.exp(decision) /
                          (np.exp(-decision) + np.exp(decision)))
expected_proba = np.c_[1 - expected_proba_class_1, expected_proba_class_1]
np.testing.assert_allclose(proba, expected_proba)
```

Basically you only check the relationship between decision_function and predict_proba rather than some consequences further down the line.

Reply:
do you want these addressed or should we merge given the two +1? (I haven't looked at it myself). @rwolst are you still working on this?

Reply:
I agree the @lesteve regression test is simpler and will add that. Do you want to keep the old regression test as it still may be useful?